pywb/utils/loaders.py

```diff
@@ -336,6 +336,7 @@ def __init__(self, **kwargs):
         if not self.cookie_maker:
             self.cookie_maker = kwargs.get('cookie')
         self.session = None
```

Comment:
```suggestion
```

Reply:
Maybe we just remove this altogether if we're no longer using it

Reply:
oh, i didn't mean to make that change for now, it didn't help with previous issues. will just undo, good catch!